zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/client/SecurityTest.java

```diff
@@ -46,7 +46,12 @@ public final class SecurityTest {
 
   @BeforeEach
   void setUp() {
-    testCluster.start().awaitCompleteTopology();
+    final var nodes = testCluster.nodes().values();
+    nodes.forEach(
+        node -> {
+          node.start();
+        });
```

Comment:
❓ Why was this necessary? Do we need this for the other tests as well?

Reply:
The test starts two Tomcat instances (in the same JVM), one for the gateway and another for the broker. Additionally, this test configures SSL. When starting the two Tomcat instances simultaneously, one of them will fail during parsing/initializing a static map of cipher aliases (see [OpenSSLCipherConfigurationParser](https://github.com/apache/tomcat/blob/9287d3342f12d20cbdb66c11228b0f80a40a43a0/java/org/apache/tomcat/util/net/openssl/ciphers/OpenSSLCipherConfigurationParser.java)). Starting them sequentially, the first one will initialize the static map, the second won't initialize it anymore.

No, it is not necessary to do this in other tests because they don't test/configure SSL.

Reply:
Could we maybe add a reasoning as comment to that, so it is more clear in the code why we have to do this.

Reply:
Done!